mqtt-cs/src/test/java/org/apache/rocketmq/mqtt/cs/test/TestDefaultChannelManager.java

```diff
@@ -1,60 +0,0 @@
-/*
```

Comment:
'test/TestDefaultChannelManager.java' replaced by 'test/channel/TestDefaultChannelManager.java' in #45